src/plugins/discover/public/application/components/table/table.tsx

```diff
@@ -54,6 +54,19 @@ export function DocViewTable({
     setFieldsWithParents(arr);
   }, [indexPattern, hit]);
 
+  const toggleColumn = useCallback(
+    (field: string) => {
+      if (onRemoveColumn && onAddColumn && Array.isArray(columns)) {
```

Comment:
nit: we could just return early here if one of the conditions is not satisfied, so not the entire function would need to be inside an if-statement

Reply:
yes, agreed, it's really 🙈 , will improve